src/test/java/ie/cit/teambravo/cardsec/alerts/MessagingAlertDistributorTest.java

```diff
@@ -57,6 +60,23 @@ public void generateAlert_when_called_then_theMessageIsFormattedCorrectly() thro
 		assertTrue(result.toString(), result.isSuccess());
 	}
 
+	@Test
+	public void generateAlert_when_calledWithBadArguments_then_throwsException() throws Exception {
```

Comment:
"throws Exception" is probably not needed since you're catching them all

Reply:
gotta catch em all